### ISSUE TITLE
Keepalive Info problem with aastra5000

### ISSUE THREAD

**Issue by fabjal** [jitsi/jigasi#103]
Hi,
I'm trying to connect jigasi to my PBX, an Aastra (mitel) 5000, and i'm running into a problem:
All my calls (inbound or outbound) end after 32 seconds. It seems that the KEEPALIVE method of the Aastra5000 (INFO) isn't handled by jigasi. here's what wireshark shows:
Aastra5000 sends:
`request: INFO sip:<number>@<pbxdomain>:5060;transport=udp;registering_acc=<account>`
And jigasi sends back:
`Status: 501 Not implemented`
And the call ends.
It seems the keepalive method isn't handled properly by jigasi which make the call to end. Is there a setting or a modification i can make to take this method into account?
I tried searching for different options for the `net.java.sip.communicator.impl.protocol.sip.acc1403273890647.KEEP_ALIVE_METHOD` but couldn't find any references.
Thanks in advance for your help.

**Comment by damencho** [comment]
I don't think this is possible at the moment, without implementing processing of the SIP INFO method. 
Currently, we have the sip protocol able to process two types of INFO messages, one is for DTMF application/dtmf-relay and the other one is for requesting keyframes application/media_control+xml.
You can disable that keep-alive option from that PBX and leave/configure jigasi to send keep alive-messages. 
I think that returning an error for keepalive messages is normal, this means that the packet has reached the endpoint, processed it by returning something even if it is an error and there is no point of disconnecting that endpoint, it means that the endpoint is alive. I would report it and to their side to fix it. 
And any contribution implementing that processor in protocol-sip in Jitsi is welcome.

**Comment by fabjal** [comment]
All right, thanks for the info.
I'll try to see if i can disable that keep-alive option from our PBX with its support team.
However, it's managed by a service provider, so i don't know if i can get info to contribute.
Thanks again.
